src/adhocracy_frontend/adhocracy_frontend/static/stylesheets/scss/components/_forms.scss

```diff
@@ -59,6 +59,21 @@ They should all look and behave similarily.
 
 /*doc
 ---
+title: Textareas
+name: textinput.textarea
+parent: textinput
+---
+
+Textareas behave exactly like text inputs however they are resizeable when enabled in modern browsers.
+
+```html_example
+<textarea></textarea>
+<textarea disabled="disabled" placeholder="disabled"></textarea>
+```
+*/
```

Comment:
There is already a textarea in the textinput documentation. That should probably be removed.